README.md

```diff
@@ -78,7 +78,7 @@ Licensing see: https://github.com/clojj/intellij-elm/blob/master/LICENSE.txt and
 
 - Manually:
 
-  Download the [latest release](https://github.com/clojj/IntelliElm/releases/latest) and install it manually using
+  Download the [latest release](https://github.com/utiliteez/intellij-elm/releases/latest) and install it manually using
```

Comment:
I don't like the idea of linking to the releases page for a forked version of this project. I can't merge this as-is.

Reply:
Hi, fixed that